chain/network/Cargo.toml

```diff
@@ -21,7 +21,7 @@ strum = { version = "0.20", features = ["derive"] }
 near-rust-allocator-proxy = "0.3.0"
 bytesize = "1.0.1"
 conqueue = "0.4.0"
-serde = { version = "1", features = ["derive"], optional=true }
+serde = { version = "1", features = ["derive", "rc", "alloc"], optional=true }
```

Comment:
why do we need the rc/alloc? do we (somehow) change how this is serialized?

Reply:
@mm-near Yes. `Edge` uses `BorshSerialize` decorator, which doesn't work on `Arc` by default, unless you enabled both "rc", and "alloc". `BorshSerialize` is designed to not require `std` to work. They have chosen not to include `std` by default.